docs/contributing/building.rst

```diff
@@ -1,10 +1,12 @@
 .. _building:
 
+If you haven't done so already, please clone the katana ( [open source](https://github.com/KatanaGraph/katana) / [enterprise](https://github.com/KatanaGraph/katana-enterprise) ) repo before you start building. 
+
 ========
 Building
 ========
 
-If you are building katana-enterprise make sure to also read the :doc:`enterprise build addenda <index>`.
+If you are building katana-enterprise make sure to also read the :doc:`enterprise build addenda <index>`, available [here](https://docs.k9h.dev/0.1.99.dev+3.8.04ffe8.b7efb9/contributing/index.html#conda)
```

Comment:
In general, use an internal references rather than hyperlinks as versions change and Sphnix checks for dangling links.

Specially, doesn't the `enterprise build addenda` link already point to the page you are talking about?

Reply:
Roger that. I tried clicking on that link but it didn't take me anywhere